guides/common/modules/con_configuring-server.adoc

```diff
@@ -7,4 +7,7 @@ This method is performed by running the installation script with one or more com
 The command options override the corresponding default initial configuration options and are recorded in the {Project} answer file.
 You can run the script as often as needed to configure any necessary options.
 
-NOTE: Depending on the options that you use when running the {Project} installer, the configuration can take several minutes to complete.
+[NOTE]
+====
+Depending on the options that you use when running the {Project} installer, the configuration can take several minutes to complete.
```

Comment:
I see this for the second time now. two things come to my mind: drop the note altogether, because every "foreman-installer" run can take minutes to complete; or move to a snippet?

I lean towards dropping this note because configuring Foreman always takes time. Otherwise, we'd have to mention this with every foreman-installer command.

Reply:
+1 to dropping the note in both places. If a specific option caused the config to take longer to complete, it would be worth mentioning. But this text is so generic ("Depending on the options...") that it is not very helpful.